pkg/network/usm/monitor.go

```diff
@@ -81,30 +84,26 @@ func NewMonitor(c *config.Config, connectionProtocolMap, sockFD *ebpf.Map, bpfTe
 		// capture error and wrap it
 		if err != nil {
 			state = NotRunning
-			err = fmt.Errorf("could not instantiate http monitor: %w", err)
+			err = fmt.Errorf("could not instantiate USM monitor: %w", err)
```

Comment:
USM monitor? not sure I understood the error

Reply:
Changed. Hopefully it is clearer/more in tune with what it really means to fall in this error case.